tests/basic/basic_test.go

```diff
@@ -14,6 +14,10 @@ import (
 	. "github.com/portworx/torpedo/tests"
 )
 
+const (
+	defaultVstate = 1
```

Comment:
`defaultVstate` constant should not be added

Reply:
removed it